crt_portal/cts_forms/migrations/0101_constant_writer_form_letter.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.2.18 on 2021-02-24 17:50
+
+from django.db import migrations
+
+def add_constant_writer_letters(apps, schema_editor):
+    ResponseTemplate = apps.get_model('cts_forms', 'ResponseTemplate')
+    subject = 'Response: Your Civil Rights Division Report - {{ record_locator }} from {{ section_name }} Section'
+    ResponseTemplate.objects.create(
+        title='CRT - Constant Writer',
+        subject=subject,
+        body="""
+{{ addressee }},
+
+You contacted the Department of Justice on {{ date_of_intake }}. Your report number is {{ record_locator }}.  We previously received similar correspondence from you concerning this matter and we responded to that inquiry.  
```

Comment:
Removing a double-space after the period to be consistent with the rest of the letter.

```suggestion
You contacted the Department of Justice on {{ date_of_intake }}. Your report number is {{ record_locator }}. We previously received similar correspondence from you concerning this matter and we responded to that inquiry.  
```